CHANGELOG.md

```
@@ -1,73 +1,76 @@
 # Prowide Core - CHANGELOG
 
-#### 9.4.19 - SNAPSHOT
+#### 10.1.18 - SNAPSHOT
```

Markdownlint

Expected: h2; Actual: h4
Heading levels should only increment by one level at a time

(MD001, heading-increment)